parsers/test/test_OPENNEM.py

```diff
@@ -13,7 +13,7 @@ def test_process_solar_rooftop(self):
             start="2021-01-01 00:00:00+00:00",
             end="2021-01-01 01:00:00+00:00",
             freq="5T",
-            closed="left",
+            inclusive="left",
```

Comment:
Are the arguments for inclusive the same as closed?

Reply:
Yes, this is what the documentation says

```
    closed : {None, 'left', 'right'}, optional
        Make the interval closed with respect to the given frequency to
        the 'left', 'right', or both sides (None, the default).

        .. deprecated:: 1.4.0
           Argument `closed` has been deprecated to standardize boundary inputs.
           Use `inclusive` instead, to set each bound as closed or open.
    inclusive : {"both", "neither", "left", "right"}, default "both"
        Include boundaries; Whether to set each bound as closed or open.
```

The unit test which uses this code continues to succeed and it generates the same results.